disttask/framework/proto/task.go

```diff
@@ -21,7 +21,7 @@ import (
 
 // task state machine
 //  1. succeed:			pending -> running -> succeed
-//  2. failed:			pending -> running -> reverting -> reverted/revert_failed
+//  2. failed:			pending -> running -> reverting -> reverted/revert_failed, pending -> failed
```

Comment:
maybe comment on when `pending -> failed` and not go through `reverting`

Reply:
> maybe comment on when `pending -> failed` and not go through `reverting`

Ok